DocsGeneratorCLI/ConfigurationBuilder.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.IO;
+using System.Linq;
+using System.Reflection;
+
+using DocsGeneration;
+
+namespace DocsGeneratorCLI {
+  public static class ConfigurationBuilder {
+    public static Configuration BuildConfiguration(CommandArguments args) {
```

Comment:
if Configuration is a Singleton, why are we returning it here? Perhaps we need to discuss to make it a `struct` then this method makes sense.